cli/connect/commands.go

```diff
@@ -453,6 +466,14 @@ var cmdInfos = []cmdInfo{
 			),
 		),
 	},
+	cmdInfo{
+		Short: setDelimiter + " [marker]",
```

Comment:
```
> \help
---
- - |
    To get help, see the Tarantool manual at https://tarantool.io/en/doc/
    To start the interactive Tarantool tutorial, type 'tutorial()' here.

    Available backslash commands:

      \set language <language>   -- set language (lua or sql)
      \set output <format>       -- set output format (lua[,line|block] or yaml)
      \set delimiter <delimiter> -- set expression delimiter
      \help                      -- show this screen
      \quit                      -- quit interactive console
...
```

```suggestion
		Short: setDelimiter + " <marker>",
```

Reply:
Done.
But I'm not sure it would be clear how it should be used.
Square brackets usually indicate that the argument is optional.
So an empty `<delimiter>` will turns off delimiter logic, here.

Reply:
The format used by Tarantool, so let it be.